packages/dev/gui/src/3D/materials/mrdl/mrdlSliderThumbMaterial.ts

```diff
@@ -461,7 +461,7 @@ export class MRDLSliderThumbMaterial extends PushMaterial {
     /**
      * @internal
      */
-    public globaRightIndexTipPosition = new Vector4(0.0, 0.0, 0.0, 1.0);
+    public globalRightIndexTipPosition = new Vector4(0.0, 0.0, 0.0, 1.0);

```

Comment:
back-compat

Reply:
you are right!!! sorry. I totally missed the internal part. ignore me :-)
Just need to be sure all references to this internal member is changed as well.